bigtable/export_test.go

```diff
@@ -110,6 +113,9 @@ func NewIntegrationEnv() (IntegrationEnv, error) {
 	if c.Project == "" {
 		c.Project = os.Getenv("GCLOUD_TESTS_GOLANG_PROJECT_ID")
 	}
+	if c.Project2 == "" {
```

Comment:
if the user does not specify a second project, maybe we can skip or warn on the copy backup test that a second project was unspecified so it cannot run?

Reply:
Done